ToyBox/classes/MainUI/PartyEditor.cs

```diff
@@ -296,12 +296,31 @@ enum ToggleChoice {
                             UI.Label($"{prog.Experience}", UI.Width(150f));
                             UI.Space(36);
                             UI.ActionButton("Set", () => {
-                                int newXP = BlueprintRoot.Instance.Progression.XPTable.GetBonus(Mathf.RoundToInt(prog.CharacterLevel));
+                                int newXP = prog.ExperienceTable.GetBonus(Mathf.RoundToInt(prog.CharacterLevel));
                                 prog.Experience = newXP;
                             }, UI.Width(125));
                             UI.Space(23);
                             UI.Label("This sets your experience to match the current value of character level.".green());
                         }
+
+                        using (UI.HorizontalScope()) {
+                            UI.Space(100);
+                            UI.ActionToggle("Levels like a Legendary Hero",
```

Comment:
consider adding some green text describing what this feature does and limitations in this UI.